test/routes/licenceConditions.test.ts

```diff
@@ -137,7 +137,7 @@ describe('/hdc/licenceConditions', () => {
     })
   })
 
-  describe('POST /hdc/licenceConditions/:section/:bookingId', () => {
+  describe('POST /hdc/licenceConditions/standard/:bookingId', () => {
```

Comment:
Doesn't look right at line 185 below. This code:

``
licenceService.getLicence.mockResolvedValue({
          licence: { key: 'value' },
          versionDetails: { conditions_version: 1 },
        })
``

Still has `conditions_version` should that be `additional_conditions_version`?

Reply:
fixed these